services/containers/service.go

```diff
@@ -13,10 +13,18 @@ import (
 )
 
 func init() {
-	plugin.Register("containers-grpc", &plugin.Registration{
+	plugin.Register(&plugin.Registration{
 		Type: plugin.GRPCPlugin,
+		ID:   "io.containerd.containers-grpc",
```

Comment:
Well, this is a plugin that registers a containers service for GRPC, so however, that matches this naming scheme. I was thinking `io.containerd.grpc.containers`.